identity/identity.go

```diff
@@ -0,0 +1,64 @@
+package identity
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"fmt"
+)
+
+func CreateNewIdentity(path string) (string, error) {
+	keystoreManager := keystore.NewKeyStore(path, keystore.StandardScryptN, keystore.StandardScryptP)
+	account, err := keystoreManager.NewAccount("")
+	if err != nil {
+		return "", err
+	}
+
+	return account.Address.Hex(), nil
+}
+
+func GetIdentities(path string) []string {
+	keystoreManager := keystore.NewKeyStore(path, keystore.StandardScryptN, keystore.StandardScryptP)
+	var ids []string
+	for _, account := range keystoreManager.Accounts() {
+		ids = append(ids, account.Address.Hex())
+	}
+	return ids
+}
+
+// signHash is a helper function that calculates a hash for the given message that can be
+// safely used to calculate a signature from.
+//
+// The hash is calulcated as
+//   keccak256("\x19Ethereum Signed Message:\n"${message length}${message}).
+//
+// This gives context to the signed message and prevents signing of transactions.
+func signHash(data []byte) []byte {
+	msg := fmt.Sprintf("\x19Ethereum Signed Message:\n%d%s", len(data), data)
```

Comment:
Maybe it's possible to append by `[]byte` variable, instead of converting `[]byte -> string -> []byte`